x/cdp/keeper/cdp.go

```diff
@@ -305,21 +311,33 @@ func (k Keeper) GetNextCdpID(ctx sdk.Context) (id uint64) {
 // IndexCdpByOwner sets the cdp id in the store, indexed by the owner
 func (k Keeper) IndexCdpByOwner(ctx sdk.Context, cdp types.CDP) {
 	store := prefix.NewStore(ctx.KVStore(k.key), types.CdpIDKeyPrefix)
+
 	cdpIDs, found := k.GetCdpIdsByOwner(ctx, cdp.Owner)
 
 	if !found {
-		idBytes := k.cdc.MustMarshalBinaryLengthPrefixed([]uint64{cdp.ID})
+		// TODO: Is using json.Marshal fine here? Have not found a good alternative
```

Comment:
i think this is okay? maybe there are better options but I think this is pretty straightforward and prob don't have much performance issues since its just a small array.